requirements.txt

```diff
@@ -27,6 +27,6 @@ gaiohttp-websocket==0.1.1
 # c2corg_common project
 # for development use a local checkout
 # -e ../v6_common
-git+https://github.com/c2corg/v6_common.git@b8eca73
+git+https://github.com/c2corg/v6_common.git@b9a11e3
```

Comment:
should now be ca2a1be

Reply:
Right! I will make the change. Don't forget to do it in your API PR as well.